modules/hangman.js

```diff
@@ -88,21 +93,49 @@ class MtgHangman {
     }
 
     handleMessage(command, parameter, msg) {
+        const [first, ...rest] = parameter.toLowerCase().split(" ");
+
         // check for already running games
         const id = msg.guild ? msg.guild.id : msg.author.id;
-        if (this.runningGames.indexOf(id) > -1) {
-            msg.channel.send('', {embed: new Discord.MessageEmbed({
-                title: "Error",
-                description: "You can only start one hangman game every "+this.gameTime/60000+" minutes.",
-                color: 0xff0000
-            })});
+        if (id in this.runningGames) {
+            const game = this.runningGames[id];
+            // The user can !hangman guess Some Card Name
+            if (first === 'guess'){
+                const correct = this.runningGames[id].body.name.toLowerCase();
+                const guess = rest.join(' ').toLowerCase();
+                if (guess.includes(correct)){
+                    // If they're correct, pretend we guessed all the letters individually
+                    const embed = this.generateEmbed(
+                        game.body,
+                        game.difficulty,
+                        game.letters,
+                        true,
+                        true
+                    );
+                    game.message.edit('', {embed});
+                    game.collector.stop('finished');
+                    msg.react('✅');
+                }
+                else {
+                    msg.react('❎');
```

Comment:
I'd suggest adding another "line" to the hangman for each failed guess, so that you can also lose by failing too many direct guesses.

Reply:
Good idea